packages/shared/src/components/modals/ShareModal.tsx

```diff
@@ -31,9 +31,12 @@ export default function ShareModal({
   ...props
 }: ShareModalProps): ReactElement {
   const isComment = !!comment;
+  const permalink = isComment
```

Comment:
The logic before was ok. Please revert. User should be able to share link to the "share post" itself not the shared post

Reply:
We have discussed this with Sab and we came to the conclusion that the user will most likely want to share the actual article since it is unlikely you will be sharing it with other private Squad or in other social platforms where the link shared was private and inaccessible to them. But feel free to continue the discussion in the thread below.

https://dailydotdev.slack.com/archives/C03QZBF46JC/p1675073160566779?thread_ts=1675068777.979039&cid=C03QZBF46JC

Reply:
See updated discussion: https://dailydotdev.slack.com/archives/C03QZBF46JC/p1675171720757089?thread_ts=1675170884.541669&cid=C03QZBF46JC